scripts/cnode-helper-scripts/cncli.sh

```diff
@@ -794,10 +801,173 @@ cncliPTsendslots() {
   done
 }
 
+#################################
+# epochdata table load process  #
+#################################
+
+getCurrNextEpoch() {
+  getNodeMetrics
+  curr_epoch=${epochnum}
+  next_epoch=$((curr_epoch+1))
+}
+
+runCurrentEpoch() {
+  getKoiosData
+  echo "Processing current epoch: ${1}"
+  stake_param_curr="--active-stake ${active_stake_set} --pool-stake ${pool_stake_set}"
+
+  ${CNCLI} leaderlog ${cncliParams} --consensus "${consensus}" --epoch="${1}" ${stake_param_curr} |
+  jq -r '[.epoch, .epochNonce, .poolId, .sigma, .d, .epochSlotsIdeal, .maxPerformance, .activeStake, .totalActiveStake] | @csv' |
+  sed 's/"//g' >> "$tmpcsv"
+}
+
+runNextEpoch() {
```

Comment:
This function should check that we have crossed into stability window like the normal leaderlog function. Else calculating values for next epoch wont be possible, or add invalid values.

Reply:
Hello @Scitz0 
I added stability window check as requested, to be pushed yet.
Just as question, couldn't the condition simply be: "if blocklog table has no entries for next epoch, then don't run runNextEpoch else run it?

Reply:
I guess this subcommand is mostly for backfill and that normal leaderlog command handle current/next. So in my view, maybe current and next shouldn't be touched at all?